src/main.rs

```diff
@@ -619,14 +614,20 @@ fn main() {
                                         &script_loader,
                                         &map,
                                         character_information,
-                                        player_position,
+                                        Vector2::new(0, 0),
                                         client_tick,
                                     );
                                     let player = Entity::Player(player);
 
                                     player_camera.set_focus_point(player.get_position());
```

Comment:
This line specifically is already making me a bit worried.

Reply:
Kind of answered in the general comment but the screen does flash at 0,0 and the moves the player to the correct position. I proposed a possible solution to it but I am open to other suggestions on how to make it better